zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js

```diff
@@ -963,8 +963,73 @@ function ParagraphCtrl($scope, $rootScope, $route, $window, $routeParams, $locat
     }
   };
 
+  // ref: https://github.com/ajaxorg/ace/blob/5021d0193d9f2bba5a978d0b1d7a4f73d18ce713/lib/ace/autocomplete.js#L454
+  const completionSupportWithoutBackend = function(str) {
```

Comment:
This function is copied from the above link, and the only difference between this and below function is 
```
if (item.meta !== '') {
  return false;
}
```